src/celeritas/io/ImportParameters.hh

```diff
@@ -114,6 +114,8 @@ struct ImportTransParameters
     ImportLoopingMap looping;
     //! Maximum number of substeps in the field propagator
     int max_substeps{1000};
+    //! Optional global "step count" cutoff (use with care!): ignored if zero
+    int max_steps{0};
```

Comment:
Yeahhhhh so the reason I did this is a little wacky, and probably should wait for future disccussion... whether we want to make the "import data" more Celeritas-centric and less geant-centric, making it the main way to provide input to our classes. Otherwise we get this:
```c++
    static std::shared_ptr<SimParams> from_import(ImportData const&,
                                                  SPConstParticles,
                                                  size_type max_steps,
                                                  size_type max_field_substeps);
```
which is error prone.

So maybe we need `SimParams::Input::from_import` and supplement it with extra data?

Reply:
For now I've implemented `SimParams::Input::from_import`.

Reply:
Oh I see your point. I like the latest implementation. And agree about the `Import` discussion. Not sure what makes more sense. Maybe a celeritas-centric will allow for expanding the importing bit to have tailored parameters for different experiments.